tools/reviewer/README.md

```diff
@@ -39,6 +39,13 @@ define a default port for each Reviewer instance in the global registry.
 * Run `./build.sh` script and after that `./test.sh` script
 * Delete repo that was cloned in the first step (`/path/to/startup-os-folder/startup-os`)
 
+## Get a Firebase token to your local machine
+After all steps from [How to install aa tool](#how-to-install-aa-tool) please do this:
```

Comment:
`s/After all steps from/After completing the steps in `
`s/please do this/do the following`

Reply:
Done.